packages/chains/deployments/zkevm.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "zkevm",
+  "chainId": "1101",
+  "contracts": {}
```

Comment:
where are the deployed contracts addresses/files? I saw the AddressesProvider is deployed at 0x27aA55A3D55959261e119d75256aadAB79aE897C